database/schema.dot

```diff
@@ -21,70 +21,70 @@
-"salaries" [label=<<FONT FACE="Helvetica"><TABLE BORDER="0" CELLBORDER="1" CELLPADDING="4" CELLSPACING="0"><TR><TD><B><FONT POINT-SIZE="16">salaries</FONT></B></TD></TR><TR><TD ALIGN="LEFT" PORT="id"><u><FONT>id</FONT></u> <FONT> [INTEGER]</FONT> NOT NULL</TD></TR><TR><TD ALIGN="LEFT" PORT="created_at"><FONT>created_at</FONT> <FONT> [TIMESTAMP]</FONT> NOT NULL</TD></TR><TR><TD ALIGN="LEFT" PORT="created_by"><FONT>created_by</FONT> <FONT> [INTEGER]</FONT></TD></TR><TR><TD ALIGN="LEFT" PORT="is_fiscal_year"><FONT>is_fiscal_year</FONT> <FONT> [BOOLEAN]</FONT> NOT NULL</TD></TR><TR><TD ALIGN="LEFT" PORT="last_updated_at"><FONT>last_updated_at</FONT> <FONT> [TIMESTAMP]</FONT> NOT NULL</TD></TR><TR><TD ALIGN="LEFT" PORT="last_updated_by"><FONT>last_updated_by</FONT> <FONT> [INTEGER]</FONT></TD></TR><TR><TD ALIGN="LEFT" PORT="officer_id"><FONT>officer_id</FONT> <FONT> [INTEGER]</FONT></TD></TR><TR><TD ALIGN="LEFT" PORT="overtime_pay"><FONT>overtime_pay</FONT> <FONT> [DOUBLE PRECISION]</FONT></TD></TR><TR><TD ALIGN="LEFT" PORT="salary"><FONT>salary</FONT> <FONT> [DOUBLE PRECISION]</FONT> NOT NULL</TD></TR><TR><TD ALIGN="LEFT" PORT="year"><FONT>year</FONT> <FONT> [INTEGER]</FONT> NOT NULL</TD></TR></TABLE></FONT>>];
+"salaries" [label=<<FONT FACE="Helvetica"><TABLE BORDER="0" CELLBORDER="1" CELLPADDING="4" CELLSPACING="0"><TR><TD><B><FONT POINT-SIZE="16">salaries</FONT></B></TD></TR><TR><TD ALIGN="LEFT" PORT="id"><u><FONT>id</FONT></u> <FONT> [INTEGER]</FONT> NOT NULL</TD></TR><TR><TD ALIGN="LEFT" PORT="created_at"><FONT>created_at</FONT> <FONT> [TIMESTAMP]</FONT> NOT NULL</TD></TR><TR><TD ALIGN="LEFT" PORT="created_by"><FONT>created_by</FONT> <FONT> [INTEGER]</FONT></TD></TR><TR><TD ALIGN="LEFT" PORT="is_fiscal_year"><FONT>is_fiscal_year</FONT> <FONT> [BOOLEAN]</FONT> NOT NULL</TD></TR><TR><TD ALIGN="LEFT" PORT="last_updated_at"><FONT>last_updated_at</FONT> <FONT> [TIMESTAMP]</FONT> NOT NULL</TD></TR><TR><TD ALIGN="LEFT" PORT="last_updated_by"><FONT>last_updated_by</FONT> <FONT> [INTEGER]</FONT></TD></TR><TR><TD ALIGN="LEFT" PORT="officer_id"><FONT>officer_id</FONT> <FONT> [INTEGER]</FONT></TD></TR><TR><TD ALIGN="LEFT" PORT="overtime_pay"><FONT>overtime_pay</FONT> <FONT> [DOUBLE_PRECISION]</FONT></TD></TR><TR><TD ALIGN="LEFT" PORT="salary"><FONT>salary</FONT> <FONT> [DOUBLE_PRECISION]</FONT> NOT NULL</TD></TR><TR><TD ALIGN="LEFT" PORT="year"><FONT>year</FONT> <FONT> [INTEGER]</FONT> NOT NULL</TD></TR></TABLE></FONT>>];
```

Comment:
The only difference between the old and new schema file seems to be that `DOUBLE PRECISION` is now `DOUBLE_PRECISION`. I think this might have been caused by an old sqlalchemy package version in my env.  I just rebuilt the diagram locally and also got `DOUBLE_PRECISION`.

Would it be possible to generate the schema and diagram within docker so we don't have to worry about package/OS differences?

Reply:
Hmmm, that would create less problems. I'm here for it.

Reply:
Done: https://github.com/lucyparsons/OpenOversight/pull/1177